torch/_torch_docs.py

```diff
@@ -4404,6 +4404,48 @@ def merge_dicts(*dicts):
             [1.0311, 0.3901, 0.5049]])
 """)
 
+add_docstr(torch.movedim,
+           r"""
+movedim(input, source, destination) -> Tensor
+
+Move `source` dim/s of the :attr:`input` to position determined by `destination`.
```

Comment:
Suggestion:

"Moves the dimension(s) of :attr:`input` at the position(s) in :attr:`source` to the position(s) in :attr:`destination`. 
Other dimensions of :attr:`input` that are not explicitly moved remain in their original order and appear at the positions not specified in :attr:`destination`."

Reply:
Thanks! I wasn' t sure how to phrase it better.